float32 hip_pitch
float32 knee
float32 calf
